omnibus/Gemfile

```diff
@@ -1,7 +1,7 @@
 source "https://rubygems.org"
 
 gem "omnibus", git: "https://github.com/chef/omnibus.git"
-gem "omnibus-software", git: "https://github.com/chef/omnibus-software.git"
+gem "omnibus-software", git: "https://github.com/chef/omnibus-software.git", branch: "nokogiri"
```

Comment:
I presume you'll fix this before merge after https://github.com/chef/omnibus-software/pull/682 is merged.